src/app/components/SubjectHeading/Search/AutosuggestItem.jsx

```diff
@@ -21,17 +21,10 @@ const AutosuggestItem = (props) => {
         to={path}
       >
         {
-          subjectComponent ?
-            <div className="autosuggest component">
-              <span>{item.label}</span>
-              <span className="aggregateBibCount">{item.aggregate_bib_count} titles</span>
-            </div>
-            :
-            <div className="autosuggest subject">
-
-              <span><em>Subject:</em> {item.label}</span>
-              <div className="aggregateBibCount">{item.aggregate_bib_count} titles</div>
-            </div>
+          <div className={`autosuggest ${subjectComponent ? 'component' : 'subject'}`}>
+            <span>{subjectComponent ? null : (<em>Subject: </em>) }{item.label}</span>
```

Comment:
Maybe the `em` should not be underlined at all, based on the screenshot in the JIRA ticket?

Reply:
Good catch! Fixed.